int32 slave_no
robot_io/RegisterData[] output_data
robot_io/RegisterInfo[] input_info
int32 action
int32 WRITE=1
int32 READ=2
int32 WRITE_AND_READ=3
---
robot_io/RegisterData[] input_data